source/sass/type.scss

```diff
@@ -232,29 +232,29 @@ a {
   font-style: italic;
   letter-spacing: 0;
 
-  @include scaleText(10px, 12px);
+  @include scaleText(12px);
 }
 
 .body-link {
   letter-spacing: 0;
   line-height: 1.5;
 
-  @include scaleText(14px, 16px);
+  @include scaleText(16px);
 }
 
 .link-small {
   letter-spacing: 0;
   line-height: 1.33;
   font-weight: 900;
 
-  @include scaleText(10px, 12px);
+  @include scaleText(12px);
 }
 
 .link-small-tags {
   letter-spacing: 0;
   line-height: 1.33;
 
-  @include scaleText(10px, 12px);
+  @include scaleText(12px);
 }
 
 .cta-link {
```

Comment:
ah I got clarification from @taisdesouzalessa on Slack - not sure if you want to make this change here but we'd like to make some tweaks to cta-link, see https://github.com/mozilla/network/issues/273#issue-220022590 (near the bottom) for detail